<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CfgTabPageUserOfxUi</class>
 <widget class="QWidget" name="CfgTabPageUserOfxUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ofx User Settings</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="Q3GroupBox" name="groupBox7">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Bank Settings</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="textLabel4_4">
        <property name="text">
         <string>FID</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="textLabel4_4_2">
        <property name="text">
         <string>ORG</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="orgEdit">
        <property name="whatsThis">
         <string>&lt;qt&gt;
Freely selectable name for your account. This field is not used by AqOfxConnect.
&lt;/qt&gt;</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="brokerEdit">
        <property name="whatsThis">
         <string>&lt;qt&gt;
Freely selectable name for your account. This field is not used by AqOfxConnect.
&lt;/qt&gt;</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="textLabel2">
        <property name="text">
         <string>Broker Id</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLineEdit" name="fidEdit">
          <property name="whatsThis">
           <string>&lt;qt&gt;
Freely selectable name for your account. This field is not used by AqOfxConnect.
&lt;/qt&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="fidButton">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Q3GroupBox" name="groupBox9">
     <property name="title">
      <string>Connection Settings</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="textLabel4_2">
        <property name="text">
         <string>Server URL</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLineEdit" name="urlEdit">
          <property name="whatsThis">
           <string>&lt;qt&gt;
Freely selectable name for your account. This field is not used by AqOfxConnect.
&lt;/qt&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="testUrlButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Test</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Q3GroupBox" name="groupBox8">
     <property name="title">
      <string>Server Capabilities</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0" colspan="2">
       <layout class="QHBoxLayout">
        <item>
         <widget class="QCheckBox" name="accountListCheck">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Supports Account List Download</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="getAccountsButton">
          <property name="text">
           <string>Get Accounts</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="statementCheck">
        <property name="text">
         <string>Supports Statement Download</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="investmentCheck">
        <property name="text">
         <string>Supports Investments</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="billPayCheck">
        <property name="text">
         <string>Supports Bill Pay</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Q3GroupBox" name="groupBox5">
     <property name="title">
      <string>Server Options</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="emptyBankIdCheck">
        <property name="toolTip">
         <string>Send empty bank id even if one is set</string>
        </property>
        <property name="whatsThis">
         <string>&lt;qt&gt;
Use this if your bank does not allow sending a bank id (even if there is such an id).
&lt;/qt&gt;</string>
        </property>
        <property name="text">
         <string>Send Empty Bank Id</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="emptyFidCheck">
        <property name="toolTip">
         <string>Send empty FID even if one is set</string>
        </property>
        <property name="whatsThis">
         <string>&lt;qt&gt;
Use this if your bank does not allow sending a FID (even if there is such an id).
&lt;/qt&gt;</string>
        </property>
        <property name="text">
         <string>Send Empty FID</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="forceSsl3Check">
        <property name="toolTip">
         <string>Force use of  SSL v3</string>
        </property>
        <property name="whatsThis">
         <string>&lt;qt&gt;
Use this if you have problems connecting to your bank.
&lt;/qt&gt;</string>
        </property>
        <property name="text">
         <string>Force SSLv3</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="sendShortDateCheck">
        <property name="toolTip">
         <string>Force use of  SSL v3</string>
        </property>
        <property name="whatsThis">
         <string>&lt;qt&gt;
Use this if you have problems connecting to your bank.
&lt;/qt&gt;</string>
        </property>
        <property name="text">
         <string>Send Short Date</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Q3GroupBox" name="groupBox4">
     <property name="title">
      <string>Expert Settings</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="textLabel5">
        <property name="text">
         <string>APPID</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="textLabel5_2">
        <property name="text">
         <string>APPVER</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="appIdEdit">
        <property name="whatsThis">
         <string>&lt;qt&gt;
Freely selectable name for your account. This field is not used by AqOfxConnect.
&lt;/qt&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="appVerEdit">
        <property name="whatsThis">
         <string>&lt;qt&gt;
Freely selectable name for your account. This field is not used by AqOfxConnect.
&lt;/qt&gt;</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="textLabel5_2_2">
        <property name="text">
         <string>Header Version</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="headerVerEdit">
        <property name="whatsThis">
         <string>&lt;qt&gt;
Freely selectable name for your account. This field is not used by AqOfxConnect.
&lt;/qt&gt;</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="textLabel5_2_2_2">
        <property name="text">
         <string>CLIENTUID</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="clientUidEdit">
        <property name="whatsThis">
         <string>&lt;qt&gt;
Freely selectable name for your account. This field is not used by AqOfxConnect.
&lt;/qt&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="spacer13">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>21</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3GroupBox</class>
   <extends>QGroupBox</extends>
   <header>Qt3Support/Q3GroupBox</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>fidEdit</tabstop>
  <tabstop>fidButton</tabstop>
  <tabstop>orgEdit</tabstop>
  <tabstop>brokerEdit</tabstop>
  <tabstop>urlEdit</tabstop>
  <tabstop>testUrlButton</tabstop>
  <tabstop>accountListCheck</tabstop>
  <tabstop>getAccountsButton</tabstop>
  <tabstop>statementCheck</tabstop>
  <tabstop>investmentCheck</tabstop>
  <tabstop>billPayCheck</tabstop>
  <tabstop>emptyBankIdCheck</tabstop>
  <tabstop>sendShortDateCheck</tabstop>
  <tabstop>emptyFidCheck</tabstop>
  <tabstop>forceSsl3Check</tabstop>
  <tabstop>appIdEdit</tabstop>
  <tabstop>appVerEdit</tabstop>
  <tabstop>headerVerEdit</tabstop>
  <tabstop>clientUidEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
